### PR TITLE
Add support for serializable field values

### PR DESCRIPTION
**Pull request by jacksonhardaker**
- **add `fast-json-stringify`**
- **wrap log function to accept serializable field values**
- **build dist package.json**

### CODE REVIEW

File: packages/react-native/project.json [hunk 1]
```diff
@@ -27,10 +25,10 @@
           "cp -r packages/react-native/README.md dist/react-native",
           "cp -r LICENSE.package dist/react-native/LICENSE",
           "cp -r packages/react-native/app.plugin.js dist/react-native",
-          "cp packages/react-native/package.dist.json dist/react-native/package.json",
+          "jq '{name, version, description, author, license, homepage, bugs, repository, main: (.main | sub(\"^dist/\"; \"\") + \".js\"), module: (.module | sub(\"^dist/\"; \"\") + \".js\"), \"react-native\": (.main | sub(\"^dist/\"; \"\") + \".js\"), types: (.types | sub(\"^dist/\"; \"\")), codegenConfig, dependencies, peerDependencies}' packages/react-native/package.json > dist/react-native/package.json",
```
Comment (jacksonhardaker) [inline packages/react-native/project.json:28]: Considered making this a script, but leaving it inline for now

File: packages/react-native/project.json [hunk 2]
```diff
@@ -14,9 +14,7 @@
     },
     "TODOrelease": {
```
Comment (snowp) [inline packages/react-native/project.json:15]: Would be nice to not have this be TODO and maybe wire it up to the release action that fz added for electron

File: packages/react-native/project.json (hunk 1 above)
Comment (snowp) [reply packages/react-native/project.json:28]: what a monster

**Review by snowp — APPROVED** [review]

File: packages/react-native/project.json (hunk 2 above)
Comment (jacksonhardaker) [reply packages/react-native/project.json:15]: Yeah, agreed. Might take a look at that this week.

**Pull request merged**